src/blog.rs

```diff
@@ -82,6 +85,26 @@ impl Blog {
 
       let (idx, lang, slug) = parse_file_name(file_name)?;
 
+      let body = if meta.format.is_none() || meta.format == Option::Some("md".to_string()) {
```

Comment:
Please change the default to md and not rst otherwise we have to change everything we've published so far.